#165413075 user can follow and unfollow each other

Kabalisa opened the pull request:
Description
=======
This feature is about allowing an authenticated user to follow an author on authors haven.

Type of change
=======
N/A

How has it been tested
=======
- pull this branch
- run ```npm install```
- run ```npm start``` to start our react app
-login into authorshaven
-choose an article written by another author
-follow the author
- unfollow the author
-navigate to your profile
- Click on either the following or followers header
- a page showing your followers and who you follow will appear
- there you can follow and unfollow authors.

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[#165413075](https://www.pivotaltracker.com/story/show/165413075)

AJAkimana commented on src/components/follow/follow.jsx:
```diff
@@ -0,0 +1,128 @@
+                  </div>
+                ))
+              ) : (
+                <div>you do not follow anyone</div>
```
Start with a capital letter sentences

AJAkimana commented on src/components/follow/follow.jsx:
```diff
@@ -0,0 +1,128 @@
+                  </div>
+                ))
+              ) : (
+                <div>you have no followers</div>
```
Start with a capital letter

AJAkimana commented on src/components/follow/follow.jsx:
```diff
@@ -0,0 +1,128 @@
+    return (
+      <div>
+        <NavBar />
+        <div className='Follow'>
```
Put a title on this page that indicates users where they are

Kabalisa replied:
@AJAkimana. the titles are already there. there is one above followers and another above following.

AJAkimana reviewed (commented):
Good work @Kabalisa, Consider the inline comments

jabichris reviewed (approved):
Nice work

Kabalisa committed:
feat(follow): user follow and unfollow authors

- user can follow and unfollow an author while reading their articles
- user can if they follow a user when reading their article
-user can follow and unfollow a user from their profile
-user can see who they follow and who follows them from their profile

[Delivers #165413075]

AJAkimana reviewed (approved):
Good work @Kabalisa
